contents/handbook/small-teams/marketing/index.mdx

```diff
@@ -13,9 +13,31 @@ hideAnchor: false
 
 <Mission />
 
-## Q4 2023 Goals
-
-<Objectives />
```

Comment:
Shouldn't all this content go in the objectives file?

Reply:
Yeah but it adds minor friction and I saw other teams don't do this

Reply:
I've a feeling not doing this breaks the careers pages

Reply:
Did it properly